tests/proc/tunderscore.nim

```diff
@@ -0,0 +1,28 @@
+discard """
+  targets: "c cpp js"
+"""
+
```

Comment:
does it also help with https://github.com/nim-lang/Nim/issues/13804 ?

Reply:
no